src/pages/index.html

```diff
@@ -207,9 +207,54 @@ <h3>David Johnson</h3>
 
 
     </main>
-    
-    
 
+    <!-- ----footer---- -->
+  <div class="wave-container">
+    <div class="wave wave1"></div>
+    <div class="wave wave2"></div>
+    <div class="wave wave3"></div>
+  </div>
+  <footer>
+    <div class="left-footer">
+      <h3>Student <span>Zynergy</span></h3>
+      <p>A grand students hub of keshav Mahavidyalaya for collaborating and improving
+      </p>
+      <span>
+        <p><i class="fa-regular fa-envelope"></i> <a href="mailto:solveeaseofficial@example.com">solveeaseofficial@gmail.com</a></p>
+      </span>
+    </div>
+    <div class="partitionLine"></div>
+    <div class="center-footer">
+      <h3>Useful Links</h3>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="contact.html">Contact</a></li>
+        <li><a href="About">About</a></li>
+        <li><a href="About">Help</a></li>
+        <li><a href="privacy_policy.html">Privacy policy</a></li>
+        <li><a href="terms_and_conditions.html">Terms and conditions</a></li>
+      </ul>
+    </div>
+    <div class="partitionLine"></div>
+    <div class="right-footer">
+      
+      <div class="socialLinks">
+        <h4>Follow Us</h4>
+        <a href="https://github.com/solve-ease/community-website"><i class="fa-brands fa-github"></i></a>
+        <a href="https://www.linkedin.com/company/solve-ease/posts/?feedView=all"><i class="fa-brands fa-linkedin"></i></a>
+        <a href=""><i class="fa-brands fa-square-instagram"></i></a>
```

Comment:
add href "#" instead of blank it will reduce score in lighthouse SEO analysis. 
Consider next time.